contracts/contracts/extensions/registry/EASProjectRegistry.sol

```diff
@@ -0,0 +1,121 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.20;
+
+import { Ownable } from "@openzeppelin/contracts/access/Ownable.sol";
+
+import { IEAS } from "../../interfaces/IEAS.sol";
+
+/// @title EASProjectRegistry
+/// @notice This contract is a simple registry of projects
+/// @dev This does not constrain the number of projects
+/// which might be > vote options
+/// @dev it does not prevent duplicate projects from being
+/// added as projects
+/// @notice It allows the attestation owners to change their own address
+contract EASProjectRegistry is Ownable(msg.sender) {
+  /// @notice a Structure holding the attestationId and the projectAddress
+  /// for each approved project
+  struct EASProject {
+    bytes32 attestationId;
+    address projectAddress;
```

Comment:
is it actually the `payoutAddress`?

Reply:
Yes, but wanted to keep the name more general vs using payout 🤔